### ISSUE TITLE
Docs

### ISSUE THREAD

**Issue by FichteFoll** [agibsonsw/PrintHtml#3]
Please, re-structure your docs. Or at least link them.

You are currently using the Wiki plugin (?)  to document your package's behaviour. However, it is hard to follow them as of now since you don't use any Markdown elements (except a code block once). Please, use Markdown. It makes text readable and allows you to highlight important aspects. Furthermore, it allows you to embed the screenshots you've uploaded anyway.

This also applies to your `README`. If you want to continue documentation with Wikis (nothing wrong with that) please at least link them in the readme. I almost died from walls of text when I found your forum thread and I was like "Where do I find how this actually works? What does it do? What about the output, how is it like?" and so on.

Just check a few other packages and their repositories here on github to understand how others use these features and how you can benefit from them just as well.

**Comment by agibsonsw** [comment]
Hello, and I apologise for the confusion - I'm still learning how to use GitHub.

I'm now using PrintHtmlFullest.rtf as a single, complete, guide. I did try previously to embed/link to an image/screenshot, but couldn't get it working. (I think I need to provide a full path for this to work.)

I'll tidy it: I might delete some of the wikis which are out of date; and update the readme to give a better overview.

I can see what you're saying though :)

**Comment by FichteFoll** [comment]
Sorry for being rude, I just wanted to make that clear. ;)

Anyway, don't refuse to use Markdown. Github supports this in almost any visual comment area (like in here) and also previews these (and a few more languages of this type I don't know well). Create a file named `README.md` or similar and experiment with it. The point is that this file will be shown when you enter a repository and it will be the first for anyone to look at! (At least that's what I do.)
You can also edit it online and preview changes in the github editor. On the top right of comment text area (which I'm currently writing into) there is a link for some references.

**Comment by agibsonsw** [comment]
I've (hopefully) tidied this repo up a little. The readme is more informative and there is one main document for guidance - PrintHtmlFullest.rtf.

I've left the screenshots for the moment as they might be useful. But I might delete a couple of the older ones. I'm not sure how I can make these more directly useful, as I don't believe I can add a description to them. If anyone has a suggestion as to how these could be made more useful, please advise.

I'll close this issue for the moment, but I'm quite happy to re-open it (or start a new one) if anyone still feels it could be more informative.

[Still can't get markdown for an image to work though :(

![GitHub Logo](github.com/agibsonsw/PrintHtml/screenshots/comment_icon2.png)
Format: ![Alt Text]

**Comment by FichteFoll** [comment]
![Test image](https://github.com/agibsonsw/PrintHtml/raw/master/screenshots/comment_icon2.png)

```
![Test image](https://github.com/agibsonsw/PrintHtml/raw/master/screenshots/comment_icon2.png)
```

-- [Reference](http://daringfireball.net/projects/markdown/syntax#img)

Well, just dig into it. And preview it to yourself. I also don't know all facets of Markdown, but for example defining headers is really useful.

### Example

```
### Example
```

For example when you have several sections in your Readme, this improves structure a lot. You can even link to these headers now since Github auto-creates anchorlinks for each header.

[Anchor](#example) | only in `README.md`

```
[Anchor](#example)
```

**Comment by agibsonsw** [comment]
Thank you!

![Test image](https://github.com/agibsonsw/PrintHtml/raw/master/screenshots/comment_icon2.png)
